.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
I wonder how you managed to ignore .idea folder when pushing this project without .gitignore :)